infra/run_tests.py

```diff
@@ -0,0 +1,98 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing perepo_managerissions and
+# limitations under the License.
+"""Runs the tests associated with the changes made to the repo."""
+import os
+import sys
+
+import utils
+
+# A list of tests that should not be run due to run time.
+BLACK_LIST = ['bisector_test.py']
+
+
```

Comment:
Sorry it's clear now what I said about only running tests on changes is ambiguous.
What I meant was that if there's any change in infra/ then run tests. This makes it unlikely that we forget to run a test but it doesn't slow down oss-fuzz project devs.
How long does it take to run every test?

Reply:
5 min 49 sec

Reply:
In that case I think running all tests makes more sense.